### DISCUSSION TITLE
TRADUCCION A ESPAÑOL

### DISCUSSION THREAD

**Discussion by JorgeRTorres** [community/community#121698] [Discover: GitHub Best Practices]
No visualizo la opcion de traducir al español la pagina de github, si lo hago desde mi navegador cambia los nombres de algunos archivos que estan cargados en ingles, busco evitar eso, alguna ayuda, muchas gracias.

**Comment by gurjeetsinghvirdee** [comment]
Hey @JorgeRTorres 

![326582829-0a89e069-7e4d-45c2-9f7b-81d47914f96c](https://github.com/community/community/assets/73753957/bed8ee1b-e1cc-430b-8c29-b79c5c1f554f)

**Comment by davevad93** [comment]
Hola @JorgeRTorres , 
lamentablemente a día de hoy el único idioma de la interfaz de GitHub disponible es el inglés, lo único que puedes hacer es cambiar tu configuración de tendencias por si quieres ver contenido recomendado en español siguiendo estos pasos:

- Haz clic en tu perfil en la parte superior derecha.
- En el menú desplegable haz clic en "Settings".
- Haz clic en "Public Profile".
- Desplázate hacia abajo hasta que veas "Trending settings".
- Haz clic en "Preferred spoken language" y elige español.
- Haz clic en "Save Trending settings" para guardar la configuración.

**Comment by goldieowner17** [comment]
Hey @JorgeRTorres, in concert with our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md), we ask that  that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
